src/ApiPlatform/Resources/Feature/BulkFeatures.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License version 3.0
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0
+ */
+
+namespace PrestaShop\Module\APIResources\ApiPlatform\Resources\Feature;
+
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\ApiResource;
+use PrestaShop\PrestaShop\Core\Domain\Feature\Command\BulkDeleteFeatureCommand;
+use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureNotFoundException;
+use PrestaShopBundle\ApiPlatform\Metadata\CQRSUpdate;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Validator\Constraints as Assert;
+
+#[ApiResource(
+    operations: [
+        new CQRSUpdate(
+            uriTemplate: '/features/features/delete',
```

Comment:
Its look weird no?
Maybe we should statuate with bulk actions to unified for all entities 🤔 

And with the same approach that Google Drive API do with a `batch` endpoint:

```suggestion
        new CQRSDelete(
            uriTemplate: '/features/batch',
```

Reply:
I based my work on this:
https://github.com/PrestaShop/ps_apiresources/blob/20fb2231e1d92a0736aaf8d68fd28136810fa251/src/ApiPlatform/Resources/Attribute/BulkAttributeGroups.php#L33-L37

But your observation is absolutely right!
I'll make the change.
Thanks!

Reply:
cc @jolelievre 

I think we need to add a few rules to ADR 🤔

Reply:
@boherm

By using CQRSDelete instead of CQRSUpdate for the batch delete, I had to create the deleteBatch() method to be able to pass the $data parameter and execute the Request::METHOD_DELETE call.

Perhaps we could add this method directly here: https://github.com/PrestaShop/ps_apiresources/blob/dev/tests/Integration/ApiPlatform/ApiTestCase.php?

What do you think?
Maybe we could use a more appropriate name.